rust/src/router/plain_transport.rs

```diff
@@ -111,19 +116,99 @@ pub struct PlainTransportDump {
     pub data_consumer_ids: Vec<DataConsumerId>,
     pub recv_rtp_header_extensions: RecvRtpHeaderExtensions,
     pub rtp_listener: RtpListener,
-    pub max_message_size: usize,
+    pub max_message_size: u32,
```

Comment:
4GB for this and `sctp_max_message_size` is enough IMHO. Plus for SCTP a `int32` is provided to `set_sock_opt`.